uwrt_mars_rover_hw/src/uwrt_mars_rover_hw_drivetrain.cpp

```diff
@@ -7,6 +7,7 @@ bool UWRTRoverHWDrivetrain::init(ros::NodeHandle& /*root_nh*/, ros::NodeHandle&
   if (!param_fetched) {
     ROS_WARN_STREAM_NAMED(name_, robot_hw_nh.getNamespace()
                                      << "/joints could not be found and loaded from parameter server.");
+    // TODO fail init?
```

Comment:
seems that it probably be best to `return false` since I assume the initialization has failed if the joints could not be obtained

Reply:
i have this changed to fail on my local branch of #39